kitchen.yml

```diff
@@ -24,16 +27,12 @@ platforms:
   - name: opensuse-leap-15
   - name: macosx-10.13
     driver:
-      box: chef/macosx-10.13 # private box in Chef's Atlas account
+      box: chef/macosx-10.13  # private box in Chef's Atlas account
       provider: vmware_fusion
   - name: opensuse-leap-15
   - name: ubuntu-16.04
-  - name: ubuntu-16.04-chef-12.7
```

Comment:
You probably want to leave one here that's 12.15 so the oldest supported infra client can be tested